lib/shoryuken/manager.rb

```diff
@@ -77,7 +77,11 @@ def processor_done(queue, processor)
 
     def processor_died(processor, reason)
       watchdog("Manager#processor_died died") do
-        logger.error { "Process died, reason: #{reason}" }
+        if reason
+          logger.error { "Process died, reason: #{reason}" }
+        else
+          logger.info { "Process stopped by user" }
```

Comment:
@mariokostelac the default log level is INFO, so it's going to print out by default in the same way died with reason prints, unless someone change the log level to error. I guess if we use error for the other, we should use error for this too, WDYT?